qiling/os/linux/function_hook.py

```diff
@@ -253,6 +263,11 @@ def set_ret(self, addr):
         # X8664
         elif  self.ql.archtype== QL_ARCH.X8664:
             self.ql.mem.write(self.ql.reg.rsp, self.ql.pack(addr))
+
+        # SPARC
```

Comment:
looks like all RSP

Reply:
good catch 👍 
I think unicorn can also support sparc v8 aka "64-bit sparc" but I didn't checked if this can easily ported to qiling